src/main/java/org/ambraproject/wombat/service/DoiToJournalResolutionService.java

```diff
@@ -77,15 +78,18 @@ public String getJournalKeyFromDoi(String doi, Site site) throws IOException {
     Map<String, ?> results = solrSearchApi.search(explicitDoiSearchQuery, site);
     List<Map<String, ?>> docs = (List<Map<String, ?>>) results.get("docs");
 
-    List<String> journalKeys = null;
-    if (docs.size() == dois.size()) {
-      journalKeys = docs.stream().map(data -> {
-          return (String) data.get("journal_key");
-      }).collect(Collectors.toList());
-    } else {
-      log.warn("incorrect solr result count " + docs.size() + " != " + dois.size()
-          + " for query=" + explicitDoiSearchQuery.getQuery());
-    }
+    Map<String, String> keysMap = new HashMap<String,String>();
+    docs.stream().forEach(data -> {
```

Comment:
You can just call the `forEach` on the `docs` instance directly, rather than creating a **stream()** first.  For example:
```
docs.forEach(...)
```